tensorflow/lite/kernels/internal/reference/reference_ops.h

```diff
@@ -314,6 +314,34 @@ inline void LeakyRelu(const tflite::LeakyReluParams& params,
   }
 }
 
+template <typename T>
+inline void QuantizeLeakyRelu(const LeakyReluParams& params, T q_alpha,
+                              const RuntimeShape& input_shape,
+                              const T* input_data,
+                              const RuntimeShape& output_shape,
+                              T* output_data) {
+  gemmlowp::ScopedProfilingLabel label("LeakyRelu (not fused)");
+  const int flat_size = MatchingFlatSize(input_shape, output_shape);
+  for (int i = 0; i < flat_size; ++i) {
+    const int32 input_value = input_data[i] - params.input_offset;
+    if (input_value >= 0) {
+      output_data[i] = input_data[i];
+    } else {
+      const int32 alpha_value = q_alpha - params.alpha_offset;
+      const int32 unclamped_output =
+          params.output_offset + MultiplyByQuantizedMultiplierSmallerThanOneExp(
+                                     input_value * alpha_value,
+                                     params.output_multiplier,
+                                     params.output_shift);
+      const int32 quantized_min = std::numeric_limits<T>::min();
```

Comment:
Although the compiler might optimize it away, I think it's better to move this out of the for loop and maybe make them "static".

Reply:
@jianlijianli , thanks for the review, i have updated the code as per your suggestion, kindly check.

Regards
Amit